README.md

```diff
@@ -62,27 +53,17 @@ After you're finished please remove all the comments and instructions!
 
 # 📖 [your_project_name] <a name="about-project"></a>
```

Comment:
- [ ] I think it would be better to add your project name here instead of just "your_project_name"